flow-preferences/src/main/java/com/fredporciuncula/flow/preferences/Serializers.kt

```diff
@@ -1,10 +1,22 @@
 package com.fredporciuncula.flow.preferences
 
+import android.content.SharedPreferences
+
+/**
+ * Represents an abstraction for serializing custom data types to store them in [SharedPreferences]
+ *
+ * Note: For example can be used as a wrapper for json serializers like Gson, Moshi, etc.
+ */
 interface Serializer<T : Any> {
   fun deserialize(serialized: String): T
   fun serialize(value: T): String
 }
 
+/**
+ * Represents an abstraction for serializing custom nullable data types to store them in [SharedPreferences]
+ *
+ * Note: For example can be used as a wrapper for json serializers like Gson, Moshi, etc.
+ */
 interface NullableSerializer<T> {
```

Comment:
🔍 I think your explanation is already pretty clear and we could drop the note to keep it brief. There's just a `.` missing in the first sentence!